metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java

```diff
@@ -67,20 +70,27 @@ public MetafixLookupTest() {
     private static void setStubForWireMock() {
         WIRE_MOCK_SERVER.start();
 
-        final UrlPattern urlPattern = WireMock.urlPathEqualTo(RDF_PATH);
+        // stubs for RDF
+        UrlPattern urlPattern = WireMock.urlPathEqualTo(RDF_PATH);
         final String redirectToUrl = "/redirect" + RDF_PATH;
         final UrlPattern urlPatternRedirectToUrl = WireMock.urlPathEqualTo(redirectToUrl);
-
         WIRE_MOCK_SERVER.stubFor(WireMock.get(urlPattern)
             .willReturn(WireMock.temporaryRedirect(redirectToUrl)));
-
-        final String responseBody = new BufferedReader(new InputStreamReader(
-                    MetafixLookupTest.class.getResourceAsStream("." + RDF_PATH))).lines().collect(Collectors.joining("\n"));
-
+        String responseBody = loadFile(RDF_PATH);
         WIRE_MOCK_SERVER.stubFor(WireMock.get(urlPatternRedirectToUrl)
             .willReturn(WireMock.aResponse()
                 .withHeader("Content-Type", "text/turtle")
                 .withBody(responseBody)));
+
+        //stub for CSV
```

Comment:
Minor: Missing space after comment marker.

Reply:
Added a checkstyle rule an btw updated this dependency. This brought up a some [code commented out](https://github.com/metafacture/metafacture-fix/pull/316/commits/8500ac4392a8760e313013a431bfdfc63aa9ed99) which I did in part just removed and in part left it (just "fixing" the structure the new checkstyle rule demands) as these comments may have some worth. I didn't find any remarks in git commits so I leave this to you to decide. Actually I don't know if you appreciate the new checkstyle rule at all or want to dump that.

Reply:
> Actually I don't know if you appreciate the new checkstyle rule at all or want to dump that.

In fact, I don't, sorry. Exactly because it can't differentiate between comments and commented-out code. Some things just have to be left to human judgement ;)

Reply:
OK. Dumped that in #317.